assets/styles.css

```diff
@@ -1,3 +1,409 @@
-.header {
- color: red;
-}
\ No newline at end of file
+body {
+  margin: 0;
+  /* font-family: ""; */
```

Comment:
- [ ] Commented code is not a good practice in clean coding. Please avoid it.

Reply:
Done.